ui/apps/platform/src/Containers/Policies/Wizard/Form/descriptors.tsx

```diff
@@ -230,10 +230,10 @@ export type TextDescriptor = {
 
 // TODO: merge with policyConfigurationDescriptor after ROX_VERIFY_IMAGE_SIGNATURE enabled by default
 export const imageSigningCriteriaDescriptor: SignatureDescriptor = {
-    label: 'Trusted image signers',
+    label: 'Not verified by trusted image signers',
     name: imageSigningCriteriaName,
```

Comment:
Another minor thing that we can do is to adjust `imageSigningCriteriaName`(and corresponding backend constant value) to be somewhat similar to the label. We can do that either in this PR or separate

Reply:
I'll do it in a separate one.